packages/app/features/settings/PrivacyScreen.tsx

```diff
@@ -149,6 +173,23 @@ export function PrivacySettingsScreen(props: Props) {
           </YStack>
         )}
 
+        <YStack paddingHorizontal="$l" paddingTop="$2xl" gap="$xl">
+          <XStack justifyContent="space-between" alignItems="center">
+            <SizableText flexShrink={1}>
+              Disable centralized features
+            </SizableText>
+            <Switch
+              style={{ flexShrink: 0 }}
+              value={state.disableTlonInfraEnhancement}
+              onValueChange={toggleDisableTlonInfraEnhancement}
+            ></Switch>
+          </XStack>
+          <Text size="$label/s" color="$secondaryText">
+            If disabled, the app will avoid falling back to Tlon's
```

Comment:
@jamesacklin copy

Reply:
I think "centralized features" is a bit of a misnomer; I would expect this to also disable my image-uploading capabilities and maybe even my hosting account? 

Let's rephrase to:

```
Disable Tlon helpers
Your ship will always attempt to generate rich link previews locally. If disabled, the app will avoid making backup requests to Tlon's service if local generation fails.
```